src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -83,27 +88,32 @@ $all-colors: map-merge($pst-semantic-colors, $extra-semantic-colors);
   html[data-theme="#{$mode}"] {
     // check if this color is defined differently for light/dark
     @each $name in $sd-semantic-color-names {
-      $definition: map-get($all-colors, $name);
-      @if type-of($definition) == map {
+      $definition: map.get($all-colors, $name);
+
+      @if meta.type-of($definition) == map {
         @each $key, $value in $definition {
-          @if str-index($key, $mode) != null {
+          @if string.index($key, $mode) {
             // since now we define the bg colours in the semantic colours and not
             // by changing opacity, we need to check if the key contains bg and the
             // correct mode (light/dark)
-            @if str-index($key, "bg") != null {
+            @if string.index($key, "bg") {
               --sd-color-#{$name}-bg: #{$value};
+
               // create local variable
-              $bg-var: --sd-color-#{$name}-bg;
+              $bg-var: var(--sd-color-#{$name}-bg);
```

Comment:
Can someone double check this line, 103?

Reply:
I can't figure out why `$bg-var` needs to be created at all here, it's not used elsewhere AFAICT.

Reply:
I'm going to delete it then. Can't find it in Sphinx Design or Bootstrap either.